packages/legend-application-repl/src/components/repl/Form.tsx

```diff
@@ -537,12 +537,12 @@ export function FormColorPickerButton(props: {
 }
 
 export const FormDocumentation: React.FC<{
+  dataCubeStore: DataCubeStore;
   documentationKey: string;
   title?: string | undefined;
   className?: string | undefined;
-}> = ({ documentationKey, title, className }) => {
+}> = ({ documentationKey, title, className, dataCubeStore }) => {
   const application = useApplicationStore();
-  const repl = useREPLStore();
```

Comment:
Let's create a `useDataCubeStore` hook

```suggestion
const store = useDataCubeStore()
```